sql/size_utils.sql

```diff
@@ -46,22 +46,37 @@ BEGIN
                FROM (
                SELECT *, total_bytes-index_bytes-COALESCE(toast_bytes,0) AS table_bytes FROM (
                       SELECT
-                      sum(pg_total_relation_size(format('%%I.%%I', c.schema_name, c.table_name)))::bigint as total_bytes,
-                      sum(pg_indexes_size(format('%%I.%%I', c.schema_name, c.table_name)))::bigint AS index_bytes,
-                      sum(pg_total_relation_size(reltoastrelid))::bigint AS toast_bytes
+                      sum(
```

Comment:
I think this was intended to kept as is and not modified to expose compressed table information.  All compression related information is exposed only via the compression specific APIs. (Perhaps the compression usage related docs has some discussion related to it). We need a broader discussion on what these APIs should expose before merging this.